### DISCUSSION TITLE
Trying to run  on https://127.0.0.1:8992 but it insists on checking for 0.0.0.0:443

### DISCUSSION THREAD

**Discussion by square-foot** [pocketbase/pocketbase#3423] [Q&A] [answered]
I am trying to run a pocketbase website using Nginx reverse proxy  which looks at 127.0.0.1:8992  Certbot was used to specify the Letsencrypt https certificates in that nginx configuration file (i.e. I dont plan to use the https setup by pocketbase)

When I run  the following 

pocketbase serve  --http "localhost:8992" 
├─ REST API: https://LICENSE.md/api/
└─ Admin UI: https://LICENSE.md/_/
_2023/09/30 16:56:41 listen tcp 0.0.0.0:443: bind: address already in use_

The version of pocketbase I am using is 19.7 

But version 17.5 of pocketbase did not have this problem as I have another instance of pocketbase serving another site using that version via the above method ... and indeed, when I used the 17.5 version for the above, it works fine

My machine is Ubuntu 22.04 

What am I doing wrong here? Thanks in advance

**Comment by ganigeorgiev** [comment]
If you want to specify only a listening address you can use the `--http` flag, aka.:

```sh
./pocketbase serve --http="127.0.0.1:8992"
```

For more details why the domain argument was introduced you can check https://github.com/pocketbase/pocketbase/discussions/3190.

**Reply by square-foot** [reply]
I just updated my note. I tried exactly what you suggested, but it did not work.  Kindly see my updated note

**Reply by square-foot** [reply]
Both arguments  
 --http="localhost:8992" 
 or 
 --http="127.0.0.1:8992" 
 
gives same error

**Reply by ganigeorgiev** [reply]
I don't know how to reproduce it. You are probably missing something in the example.

We don't listen to the `https` port if a domain argument is not specified.
You can test it locally and confirm it.

**Reply by square-foot** [reply]
my nginx web server is running several of my websites and all of them use lets Encrypt. Can that be a problem?

**Reply by ganigeorgiev** [reply]
I'm confused. Again, if you just run `./pocketbase serve --http="127.0.0.1:8992"` this will bind to the `127.0.0.1:8992`, not `0.0.0.0:443` address. Please test it locally.

**Reply by square-foot** [reply]
I tried what you suggested.  And it seems to work the way you say when I run it locally. But I wrote a bash script for it  and thru that .bash file , it gives the same error 

The bash file contains just this line:

./pocketbase serve  --http "127.0.0.1:8992"  --origins * &


Pardon me; I may have done something quite naive! I am flummoxed why it works when I give the command on the command line but not thru the script!! I am running as root user

**Reply by ganigeorgiev** [reply, marked as answer]
You need to wrap the `*` in quotes or add `=` after the flag otherwise it will treat the `*` as an argument and not as flag value, aka.:

```sh
./pocketbase serve --http="127.0.0.1:8992" --origins="*" &
```
_Note that the default origins value is already * and the above can be skipped._

**Reply by square-foot** [reply]
Ah.... so that was the problem! Thank you very much
